webapp/components/tokenSelector/tokenList.tsx

```diff
@@ -82,11 +82,22 @@ export const TokenList = function ({
 
   const { width } = useWindowSize()
 
-  const tokensToList = tokens.filter(
-    token =>
-      token.name.toLowerCase().includes(searchText.toLowerCase()) ||
-      token.symbol.toLowerCase().includes(searchText.toLowerCase()),
-  )
+  const tokensToList = tokens
+    .filter(
+      token =>
+        token.name.toLowerCase().includes(searchText.toLowerCase()) ||
+        token.symbol.toLowerCase().includes(searchText.toLowerCase()),
+    )
+    .sort(function (a, b) {
+      if (a.symbol === 'ETH') return -1
+      if (b.symbol === 'ETH') return 1
+      // using startsWith due to .e symbol version in Hemi
+      if (a.symbol.startsWith('USDC')) return -1
+      if (b.symbol.startsWith('USDC')) return 1
+      if (a.symbol.startsWith('USDT')) return -1
+      if (b.symbol.startsWith('USDT')) return 1
+      return a.name.localeCompare(b.name)
+    })
```

Comment:
Thinking if the code is more readable if the 3 tokens are taken out of the list, then reinserted at the beginning. What do you think?

Reply:
I implemented a more readable version here [01529b5](https://github.com/hemilabs/ui-monorepo/pull/664/commits/01529b598cf931eb460e47d2327d613d3e6916eb)